crates/ln-dlc-node/src/tests/system_tests/channel_less_payment.rs

```diff
@@ -0,0 +1,154 @@
+use bitcoin::Address;
+use std::str::FromStr;
+use std::time::Duration;
+// Related issue: https://github.com/get10101/10101/issues/55.
+use crate::tests::system_tests::create_tmp_dir;
+use crate::tests::system_tests::fund_and_mine;
+use crate::tests::system_tests::has_channel;
+use crate::tests::system_tests::init_tracing;
+use crate::tests::system_tests::log_channel_id;
+use crate::tests::system_tests::setup_ln_node;
+
+#[tokio::test]
+async fn given_no_channel_with_coordinator_when_invoice_generated_then_can_be_paid_through_coordinator(
+) {
+    init_tracing();
+
+    let test_dir = create_tmp_dir("multi_hop_test");
```

Comment:
🔧 `multi_hop_test` that should probably be something like `channel_less_payment`. Although I think it doesn't matter what we provide here as an argument, as it will always be within in a random temp dir. We could as well remove that argument from the function.